bin/assets/scripts/newform.js

```diff
@@ -6,15 +6,13 @@ code.addEventListener('keydown', (event) => {
   if (event.code === 'Tab') {
     // prevents tab from being used to navigate between browser elements
     event.preventDefault();
-
-    const pos = code.selectionStart;
-    const content = code.value;
+    const { value: content, selectionStart, selectionEnd } = code;
```

Comment:
```suggestion
    const { value, selectionStart, selectionEnd } = code;
```
J'ai été con là dessus, il n'y aucun intérêt à renommer `value` en `content`.